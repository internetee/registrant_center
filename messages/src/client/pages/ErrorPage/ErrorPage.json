[
  {
    "id": "errorpage.page_title",
    "defaultMessage": "Error | EIS Registreerijaportaal"
  },
  {
    "id": "errorpage.title",
    "defaultMessage": "Lehte ei leitud"
  },
  {
    "id": "hero.link.back",
    "defaultMessage": "Tagasi"
  },
  {
    "id": "errorpage.none.message.title",
    "defaultMessage": "Kahjuks sellisele URL-le vastavat lehte ei leitud"
  },
  {
    "id": "errorpage.none.message.text",
    "defaultMessage": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, consequuntur est facere fuga illum ipsa ipsum quasi quo! Commodi consequuntur eligendi harum non ut. Architecto blanditiis dignissimos nulla placeat praesentium."
  }
]