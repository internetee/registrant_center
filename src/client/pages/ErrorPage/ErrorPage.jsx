import React from 'react';
import { FormattedMessage } from 'react-intl';
import {Icon} from 'semantic-ui-react';
import {Helmet, MainLayout, PageMessage} from '../../components';

function ErrorPage({ ui, user, history })  {
  return (
    <MainLayout ui={ui} user={user}>
      <FormattedMessage
        id="errorpage.page_title"
        defaultMessage='Error | EIS Registreerijaportaal'
      >
        {title => (
          <Helmet>
            <title>{title}</title>
          </Helmet>
        )}
      </FormattedMessage>
      <div className='main-hero'>
        <FormattedMessage
          id='errorpage.title'
          defaultMessage='Lehte ei leitud'
          tagName='h1'
        />
        <button type='button' className='back-link' onClick={ history.goBack }>
          <Icon name='arrow left' />
          <FormattedMessage
            id='hero.link.back'
            defaultMessage='Tagasi'
          />
        </button>
      </div>
      <PageMessage
        headerContent={(
          <FormattedMessage
            id="errorpage.none.message.title"
            defaultMessage="Kahjuks sellisele URL-le vastavat lehte ei leitud"
            tagName="span"
          />
        )}
        icon="times circle outline"
      >
        <FormattedMessage
          id='errorpage.none.message.text'
          defaultMessage='Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque, consequuntur est facere fuga illum ipsa ipsum quasi quo! Commodi consequuntur eligendi harum non ut. Architecto blanditiis dignissimos nulla placeat praesentium.'
          tagName='p'
        />
      </PageMessage>
    </MainLayout>
  );
}

export default ErrorPage;
