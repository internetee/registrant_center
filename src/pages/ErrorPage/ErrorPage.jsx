import React from 'react';
import { FormattedMessage } from 'react-intl';
import { MainLayout, PageMessage } from '../../components';

const ErrorPage = () => {
    return (
        <MainLayout hasBackButton titleKey="errorpage.title">
            <PageMessage
                headerContent={<FormattedMessage id="errorpage.none.message.title" />}
                icon="times circle outline"
            >
                <FormattedMessage id="errorpage.none.message.text" tagName="p" />
            </PageMessage>
        </MainLayout>
    );
};

export default ErrorPage;
