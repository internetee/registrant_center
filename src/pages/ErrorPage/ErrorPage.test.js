import 'core-js';
import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from './ErrorPage';

describe('pages/ErrorPage', () => {
    it('should render', () => {
        const component = shallow(<ErrorPage />);
        expect(component).toMatchSnapshot();
    });
});
