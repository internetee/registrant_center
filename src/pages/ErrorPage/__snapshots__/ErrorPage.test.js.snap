// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/ErrorPage should render 1`] = `ShallowWrapper {}`;
